0-ad-history-mod-package/simulation/templates/units/athen/infantry_spearman_b.xml

```diff
@@ -5,6 +5,11 @@
     <GenericName>Athenian Hoplite</GenericName>
     <SpecificName>Hoplítēs Athēnaîos</SpecificName>
     <Icon>units/athen/infantry_spearman.png</Icon>
+    <History>
+        Ancient Athens' hoplites were drawn from the third highest social class, the “Zeugitai”, people who earned or produced at least 200 medimnoi (an Ancient Greek unit of volume equal to around 50 liters) of goods per year. This was to ensure they would have the financial means to provide their own hoplite equipment, which was, for the average citizen, quite expensive.
+        It consisted of a round convex shield, about 3 feet (1 meter) in diameter, which was reinforced with copper on the outside. The main weapon was a 7- to 9-foot (around 2.5 meters)-long spear, which had an iron spearhead on one end and a spike on the other end as a counterweight. They also carried a short sword as a secondary weapon. Besides perhaps a bronze helmet, a hoplite's body armor varied between individuals, depending on what they could afford. Some wore bronze breastplates and greaves, others only leather or linen armor. In total, a hoplite's equipment weighed up to 70 pounds (about 30 kilograms).
+        In Athens, only 18- to 20-year-olds were obligated to actively take part in military training. However, should war break out, men up to the age of 60 could be called up to serve in the Athenian army. This meant Athens' huge hoplite force, at its peak over 30.000 men strong, largely consisted of militias who lacked training and experience. But still, because of the patriotic morale and of these Athenian soldiers, many willing to sacrifice their lives for the common good of Athens, under the command of skilled leaders, Athenian hoplites dominated ancient Greece.
```

Comment:
```suggestion
        In Athens, only 18- to 20-year-olds were obligated to actively take part in military training. However, should war break out, men up to the age of 60 could be called up to serve in the Athenian army. This meant Athens' huge hoplite force, at its peak over 30,000 men strong, largely consisted of militias who lacked training and experience. But still, because of the patriotic morale and of these Athenian soldiers, many willing to sacrifice their lives for the common good of Athens, under the command of skilled leaders, Athenian hoplites dominated Ancient Greece.